src/components/UserInterface/SearchBar/SearchBar.js

```diff
@@ -1,18 +1,29 @@
+import { useRef } from "react";
 import styles from "./SearchBar.module.css";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const SearchBar = (props) => {
+  const ref = useRef("");
+  // console.log(value);
+  const ClickHandler = () => {
```

Comment:
The naming for the handler should be in camelCase as per the [Javascript style guide](https://github.com/airbnb/javascript#naming--camelCase), and a preferable format is `handleEvent` so for this case `handleOnClick` 

Note: There is no "right way" to name event handler functions in React, but in the spirit of clean, consistent and readable code, there are some conventions. You can [read here ](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)on one of the common ones. 

```suggestion
  const handleOnClick = () => {
```